recipes/sentinelhub/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "sentinelhub" %}
+{% set version = "2.5.3" %}
```

Comment:
A few days ago we released a new version 2.6.0 [releases](https://github.com/sentinel-hub/sentinelhub-py/releases). The SHA from PyPI for the new version is `94b1ff8402291683a801ae6293a3982360b9b7d90657293c04e16edb55a7ed10`

Reply:
Addressed; thanks!